src/Form/CommitteeFeedMessageType.php

```diff
@@ -3,43 +3,31 @@
 namespace AppBundle\Form;
 
 use AppBundle\Committee\Feed\CommitteeMessage;
-use Symfony\Component\Form\AbstractType;
-use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
-use Symfony\Component\Form\Extension\Core\Type\SubmitType;
-use Symfony\Component\Form\Extension\Core\Type\TextareaType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class CommitteeFeedMessageType extends AbstractType
+class CommitteeFeedMessageType extends CommitteeFeedItemMessageType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder
-            ->add('subject', TextType::class, [
-                'label' => false,
-                'attr' => [
-                    'class' => 'form--full b__nudge--bottom-medium',
-                    'placeholder' => 'Entrez l\'objet de votre message',
-                ],
-            ])
-            ->add('content', TextareaType::class, [
-                'label' => false,
-                'attr' => ['placeholder' => 'Écrivez ici votre message public'],
-                'filter_emojis' => true,
-                'purify_html' => true,
-            ])
-            ->add('published', CheckboxType::class, [
-                'label' => 'Rendre ce message visible sur la page du comité',
-                'attr' => ['class' => 'form__checkbox form__checkbox--large'],
-                'required' => false,
-            ])
-            ->add('send', SubmitType::class, ['label' => 'Envoyer'])
-        ;
+        $builder->add('subject', TextType::class, [
+            'label' => false,
+            'attr' => [
+                'class' => 'form--full b__nudge--bottom-medium',
+                'placeholder' => 'Entrez l\'objet de votre message',
+            ],
+        ]);
+
+        parent::buildForm($builder, $options);
```

Comment:
Please use Form inheritance like shown here -> https://github.com/EnMarche/en-marche.fr/blob/develop/src/Form/BecomeAdherentType.php

This is not the normal way of doing inheritance in forms. Event if it's working like this right now, we don't know what sort of side effects it could do. Let's be safe.

The Form Component has its own way of managing inheritance for some reasons. 

You can do Php Inheritance for classes that are like `AbstractType` meaning that they are not a FormType. They are just a way of factorizing a bunch of code but I don't think it's a good solution either.

Really sorry about that.

See :
http://symfony.com/doc/master/form/create_form_type_extension.html#generic-form-type-extensions
https://stackoverflow.com/questions/22861044/symfony-formtype-getparent-vs-inheritance
https://github.com/symfony/symfony/issues/3879

Reply:
There are cases when we can use PHP inheritance, but this one is not. Because we don't use those types as complex service or such, we want to keep configuration and template inheritance mechanism handled by the component.
Please use `getParent()` and remove calls to `parent::xx()`.